fabulaws/library/wsgiautoscale/api.py

```diff
@@ -844,7 +844,7 @@ def update_requirements():
     # add HOME= so if there's an error, pip can save the log (Fabric doesn't
     # pass -H to sudo)
     cmd = ['HOME=%(home)s %(virtualenv_root)s/bin/pip install -q' % env]
-    cmd += [' --use-wheel']
+    cmd += [' --only-binary']
```

Comment:
```--only-binary``` requires an argument. (https://pip.pypa.io/en/stable/reference/pip_install/#install-only-binary)

Reply:
Good catch. Oddly it seemed to work okay without it, but I'm not sure what behavior it defaults to when no argument is supplied.